wp-content/themes/amplify/amplify-feed.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Single Post Template: Amplify Feed
+ * Template Name: Amplify Feed Template
+ * Description: Shows the saved links feed that can be embedded on external sites.
+ */
+?>
+
+<!DOCTYPE html>
+<!--[if lt IE 7]> <html <?php language_attributes(); ?> class="no-js ie6"> <![endif]-->
+<!--[if IE 7]>    <html <?php language_attributes(); ?> class="no-js ie7"> <![endif]-->
+<!--[if IE 8]>    <html <?php language_attributes(); ?> class="no-js ie8"> <![endif]-->
+<!--[if IE 9]>    <html <?php language_attributes(); ?> class="no-js ie9"> <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!--> <html <?php language_attributes(); ?> class="no-js"> <!--<![endif]-->
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <link rel="profile" href="https://gmpg.org/xfn/11" />
+    <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
+    <title>Amplify Embed Feed | INN Amplify Project</title>
+    <?php wp_head(); ?>
```

Comment:
Just a comment: we'll want to look at what gets enqueued here and may want to dequeue or enqueue stylesheets and scripts here as appropriate in order to make sure that the embed is as lightweight as possible.

Reply:
Right. I figured that can probably happen once we narrow down the design and then we can decide what's needed and what's not.

Reply:
Yep. I copied this note into #127, so we don't lose track of it.